bigquery/src/main/resources/interpreter-setting.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "group": "bigquery",
+    "name": "sql",
+    "className": "org.apache.zeppelin.bigquery.bigQueryInterpreter",
+    "properties": {
+      "zeppelin.bigquery.project_id": {
+        "envName": null,
+        "propertyName": "zeppelin.bigquery.project_id",
+        "defaultValue": " ",
+        "description": "Google Project ID"
+      },
+      "zeppelin.bigquery.wait_time": {
+        "envName": null,
+        "propertyName": "zeppelin.bigquery.wait_time",
+        "defaultValue": "5000",
+        "description": "Query timeout in Milliseconds"
+      },
+      "zeppelin.bigquery.max_no_of_rows": {
+        "envName": null,
+        "propertyName": "zeppelin.bigquery.max_no_of_rows",
+        "defaultValue": "100000",
+        "description": "Maximum number of rows to fetch from BigQuery"
+      }
+    }
+  }
+]
```

Comment:
You can add the above property list to `bigquery.md` as a configuration table like [this](https://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/interpreter/alluxio.html#configuration).